xds/src/main/java/io/grpc/xds/XdsServerWrapper.java

```diff
@@ -640,15 +642,16 @@ private boolean isPermanentError(Status error) {
       List<VirtualHost> virtualHosts = routingConfig.virtualHosts().get();
       if (virtualHosts == null) {
         String errorMsg = "Missing xDS routing config VirtualHosts due to RDS config unavailable.";
+        logger.log(Level.WARNING, errorMsg);
```

Comment:
No, we don't want this as per-RPC logging. We want this when the config is received. Ditto below.